doc/boot_storage_needed_info.md

```diff
@@ -0,0 +1,38 @@
+List of Information Needed About Storage Configuration
+======================================================
+
+Mount Points
+---------------
+
+- what partition is mounted to "/" ( so it can be queried )
+- what partition is mounted to "/boot" and if it is same as "/" ( so it can be queried )
+- what partition is mounted to "/boot/zipl" ( only on s390 to query )
+- what partition is mounted to "/boot/efi" ( only on efi based archs )
```

Comment:
well, for me it is what I need to know, not how :) So it is up to storage guys if there provide method like `root_partition` or `GetMountPoints` ( like now) or `mountpoint: "/"`. What is important for me is to recognize that two mount point points to same device.